src/Orchard.Web/Modules/Orchard.Users/Views/EditorTemplates/Parts/Users.SecuritySettings.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model Orchard.Users.Models.SecuritySettingsPart
+
+<fieldset>
+    <legend>@T("Security settings")</legend>
+    <div>
+        <label>@T("Life span of authentication cookies")</label>
+        @Html.TextBoxFor(m => m.AuthCookieLifeSpan, new { @class = "text medium" })
+        @Html.ValidationMessageFor(m => m.AuthCookieLifeSpan)
+        <p class="hint">@T("The time after signing in after which authentication cookies will be invalidated, in standard TimeSpan format (dd.hh:mm:ss).")</p>
+    </div>
+    <div>
+        @Html.CheckBoxFor(m => m.ShouldInvalidateAuthOnPasswordChanged)
+        <label class="forcheckbox" for="@Html.FieldIdFor(m => m.ShouldInvalidateAuthOnPasswordChanged)">@T("Invalidate authentication cookies on password change")</label>
+        @Html.ValidationMessageFor(m => m.ShouldInvalidateAuthOnPasswordChanged)
+        <p class="hint">@T("Select this flag if you want that changing a user's password will require them to sign in anew from all clients.")</p>
```

Comment:
`When checked, the user will be disconnected from all clients when their password is changed.`

Reply:
ok